CMakeLists.txt

```diff
@@ -14,11 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ~~~
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.16.3)
```

Comment:
Why 3.16.3?  It appears from the below changes that the functionality required by this file is actually supported at 3.14.0.

Reply:
True, I'd be happy settling for 3.14 I picked 3.16 based on the past precedent by [mikew-lunarg](https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/1029#issuecomment-546086192)